spec/controllers/api/v1/extensions_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+module Api
+  module V1
+    describe ExtensionsController do
+      let(:mock_course_id) { 16 }
+      let(:mock_lms_id) { 1 }
+      let(:mock_assignment_id) { 9 }
+      let(:mock_extension_id) {5}
+
+      
+
+    ### TODO: check that it's properly handling post body as well as params.
```

Comment:
general comment: how do these tests verify that (eg) the params are valid and so forth, even if the actions are just stubs right now?  in other words, when it fails with 501, how does the test know it's throwing 501 for the "right reason"? justcurious, something to discuss in meeting perhaps